src/stores/prompts.js

```diff
@@ -241,6 +241,15 @@ export const usePromptStore = defineStore('prompts', {
       }
     },
 
+    async getPromptDates() {
+      const set = new Set()
+      await this.fetchPrompts()
+      this.getPrompts?.map((prompt) => {
+        set.add(prompt?.id)
+      })
+      return Array.from(set)
+    },
+
```

Comment:
There is an issue that can be reproduced as follose:
1. Open Vue Tab in Google Chrome Dev Tools
2. Open Pinia section/Root/prompts
3. Navigate to Search page
4. Observe it is loading 6 items initially
5. When scrolling to load more, it loads all the rest available and stops fetching from firebase (that's also visible in network tab)
6. Open Create prompt dialog
7. Close dialog
8. Scroll down on the search page
9. Observe it restarts the fetching of prompts

Reply:
I will look into it

Reply:
@marclupanc Yes, I have fixed this issue, please check it.